sources/packages/backend/apps/test-db-seeding/src/main.ts

```diff
@@ -0,0 +1,36 @@
+require("../../../env_setup_apps");
+import { NestFactory } from "@nestjs/core";
+import { CleanDb } from "./clean-db/clean-db";
+import { TestDbSeedingModule } from "./test-db-seeding.module";
+import { SeedExecutor } from "./seed-executors/seed-executor";
+// Clean db command line parameter.
+const CLEAN_DB = "--task=clean-db";
+
+async function bootstrap() {
+  const app = await NestFactory.create(TestDbSeedingModule);
+
+  // Checking for CLEAN_DB parameter.
+  if (process.argv.includes(CLEAN_DB) && process.argv.length === 3) {
+    // Clean db.
+    await app.get(CleanDb).onModuleInit();
```

Comment:
I do not see a reason to have the `CleanDb` implementing the `OnModuleInit`. It seems to be a regular service with just a method to be called.
The method `app.get(CleanDb)` returns a type object, whatever method is declared in the service can be called.

Reply:
oh..ya....I was planning to check and remove the `onModuleInit` and `onApplicationBootstrap`. updating